mlir/test/Dialect/Vector/vector-emulate-narrow-type-unaligned-dynamic.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: mlir-opt --test-emulate-narrow-int="arith-compute-bitwidth=1 memref-load-bitwidth=8" --cse --split-input-file %s | FileCheck %s
+
+// CHECK: #map = affine_map<()[s0, s1] -> ((s0 * 3 + s1) floordiv 4)>
+// CHECK: #map1 = affine_map<()[s0, s1] -> ((s0 * 3 + s1) mod 4)>
+func.func @vector_load_i2(%arg1: index, %arg2: index) -> vector<3xi2> {
```

Comment:
[nit] Could you use more descriptive variable names? e.g. `arg1` -> `idx1`? It makes little difference for one test, but a gigantic one in the grand scheme of things :)